modules/validations/mutually_exclusive_tags.js

```diff
@@ -0,0 +1,46 @@
+import { t } from '../core/localizer';
+import { utilDisplayLabel } from '../util';
+import { validationIssue } from '../core/validation';
+
+export function validationMutuallyExclusiveTags(/* context */) {
+    const type = 'mutually_exclusive_tags';
+
+    // https://wiki.openstreetmap.org/wiki/Special:WhatLinksHere/Property:P44
+    const tagPairs = [
+        ['noname', 'name'],
+        ['noref', 'ref'],
+        ['noaddress', 'addr:housenumber'],
+        ['noaddress', 'addr:housename'],
+        ['noaddress', 'addr:unit'],
+        ['addr:nostreet', 'addr:street']
+    ];
+
+    const validation = function checkMutuallyExclusiveTags(entity /*, graph */) {
+
+        const pairsFounds = tagPairs.filter((pair) => {
+            return (pair[0] in entity.tags && pair[1] in entity.tags);
+        });
+
+        let issues = pairsFounds.map((pair) => {
+            return new validationIssue({
+                type: type,
+                severity: 'warning',
+                message: function(context) {
+                    let entity = context.hasEntity(this.entityIds[0]);
+                    return entity ? t.append('issues.' + type + '.message', {
+                        feature: utilDisplayLabel(entity, context.graph()),
+                        tag1: pair[0],
+                        tag2: pair[1]
+                    }) : '';
+                },
```

Comment:
In addition to specifying a `reference`, consider supplying a `dynamicFixes` with two fixes that remove either of the contradictory tags.

Reply:
I couldn't find any other translation strings using HTML and when supplying values for the `{}` replacements any HTML gets stripped. I added quotes around the name.

> consider supplying a dynamicFixes with two fixes that remove either of the contradictory tags.

I added fixes now.